packages/next-sample/pages/protected.tsx

```diff
@@ -0,0 +1,37 @@
+import Link from 'next/link';
+// eslint-disable-next-line import/no-named-as-default
```

Comment:
Is this needed?

Reply:
Yes, "next/router" is odd, not go deep into it since is sample.